docs/index.html

```diff
@@ -48,7 +48,7 @@ <h3>Supported data sources</h3>
             <div class="col-xs-2 text-right"><i class="fa fa-2x fa-bug"></i></div>
             <div class="col-xs-10">
               <p>Issues / Tasks Management</p>
-              <p class="text-justify small">Bugzilla, GitHub Issues, GitLab Issues, JIRA, Maniphest, Redmine</p>
+              <p class="text-justify small">Bugzilla, GitHub Issues, GitLab Issues, JIRA, <i>Launchpad</i>(*), Maniphest, <i>Pagure(*)</i>, Redmine <em>(*partially supported)</em></p>
```

Comment:
`partially supported` seems too vague, it could be replaced by (i) `only gathering and enrichment` or (ii) `no visualizations`. WDYT?

Reply:
Oh, thank you for the update in the data sources!

Regarding the text details, I need to review that text, at least, to ensure it looks also nice.

I'll update this ASAP. Thank you for the review!